app/scripts/views/settings/clients.js

```diff
@@ -151,7 +151,20 @@ define(function (require, exports, module) {
       return this._attachedClients.fetchClients({
         devices: true,
         oAuthApps: true
-      }, this.user);
+      }, this.user).then(() => {
+        // log the number of items
+        const numOfClients = this._attachedClients.length;
+
+        if (numOfClients === 0) {
+          this.logViewEvent('items.zero');
```

Comment:
If a user clicks `refresh`, the submit would get called an these metrics could get reemitted. Should this be a log once kind of event?